exe/PowershellShim.ps1

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env powershell
+#requires -version 2.0
+
+try {
+  $here = (Split-Path -Parent $MyInvocation.MyCommand.Path)
+  . $here\PowershellShim-Helper.ps1
+
+  $scriptArgsHash = Get-ContentAsJson ($input -join "")
+
+  Write-Debug "`nTask-shim deserialized arguments to Hashtable:`n"
```

Comment:
Do we want these `Write-Debug` lines?

Reply:
They should do nothing normally. They're maybe useful in debugging the shim, but probably not that useful.